python/README.md

```diff
@@ -19,6 +19,8 @@ git clone https://github.com/epfl-lasa/control-libraries
 bash control-libraries/source/install.sh
 
 ## install the bindings using the pip installer
+export OSQP_INCLUDE_DIR='/path/to/include/osqp' # default /usr/local/include/osqp
+export OPENROBOTS_INCLUDE_DIR='/opt/openrobots/include' # default /opt/openrobots/include
```

Comment:
```suggestion
export OSQP_INCLUDE_DIR='/path/to/include/osqp'  # default /usr/local/include/osqp
export OPENROBOTS_INCLUDE_DIR='/path/to/openrobots/include'  # default /opt/openrobots/include
```

Reply:
Why do we do two spaces before a comment now?

Reply:
Ah well that's just the PEP style in Python for inline comments, so I don't know if it really applies to shell scripts - but I feel it separates the comment content from the code content more clearly for easier reading (in my opinion). Up to you if you want to include that change. 
Mainly I wanted to highlight the `/path/to/` part to be consistent.